### PR TITLE
WINDUPRULE-616 Spring Boot Properties to quarkus rule

### PR DESCRIPTION
**Pull request by aureamunoz**

### CODE REVIEW

**Comment by aureamunoz** [comment]
Could you please take a look @jonathanvila ?

**Comment by windupgithubbot1** [comment]
Can one of the admins verify this patch?

File: rules-reviewed/quarkus/springboot/springboot-properties-to-quarkus.windup.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="springboot-properties-to-quarkus"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset gives hints to migrate from SpringBoot DI to Quarkus spring-di extension
```
Comment (m-brophy) [inline rules-reviewed/quarkus/springboot/springboot-properties-to-quarkus.windup.xml:6]: This comment should refer to properties

Comment (aureamunoz) [reply rules-reviewed/quarkus/springboot/springboot-properties-to-quarkus.windup.xml:6]: yes, absolutely. thank you, I will change it!

**Comment by m-brophy** [comment]
Thanks @aureamunoz

**Pull request merged**